### ISSUE TITLE
Possible unresolved dependency (absolute path warning) during bii find and build

### ISSUE THREAD

**Issue by spideyclick** [dimitrikourk/small3d#28]
Installing the small3d engine, but got a warning:  after downloading and initializing the project folder, I ran bii find and came across a warning:

`WARN: Block dimitrikourk/small3d has absolute paths, like: #include "dimitrikourk/small3d/resources/shaders/OpenGL33/simpleShader.vert" within same block not recommended.`
`Consider using relative path or using the [paths] config`

In addition, when I go to build the project:
`bii build`
`[ 94%] Building CXX object dimitrikourk_small3d/CMakeFiles/dimitrikourk_small3d.dir/samplegame/GameLogic.cpp.o
/home/me/Downloads/small3d-master/samplegame/GameLogic.cpp:66:50: fatal error: dimitrikourk/small3d/MathFunctions.hpp: No such file or directory`
` #include <dimitrikourk/small3d/MathFunctions.hpp>`
`                                                  ^`
`compilation terminated.`
`make[2]: *** [dimitrikourk_small3d/CMakeFiles/dimitrikourk_small3d.dir/samplegame/GameLogic.cpp.o] Error 1`
`make[1]: *** [dimitrikourk_small3d/CMakeFiles/dimitrikourk_small3d.dir/all] Error 2`
`make: *** [all] Error 2`
`ERROR: Build failed`

I'm using Linux Mint 17, so I realize my environment may be pretty altered from Debian.

I don't know if this is an issue with the code or (I'll bet more likely) my own system.  This seemed like the best place to look for help--thanks!

**Comment by dimitrikourk** [comment]
Don't worry about the warning. I tried to correct it unsuccessfully at some point and I will try to again, in the following version. It will not affect the way the block works though.

About the compilation error, it does not get reproduced on my Debian, but I am not sure it is because of Linux Mint. Could you let me know how you have used the block exactly? For example, did you clone it from GitHub, or did you add the headers to a new block? It could be that it does not work in your scenario and I need to correct something... Also please let me know which version you are using.

**Comment by spideyclick** [comment]
I do apologize, this is a case of me jumping the gun too soon!  The error, as usual, was located somewhere between the keyboard and the seat...

As far as I can tell, the problem came from me trying to clone your github repository and initialize that as a bii project.  That is not how you said to do it on the tutorial, and I think that is what gave me issues.  When I took the time to carefully read and follow your instructions without any shortcuts, wonder of all wonders, it worked flawlessly!

Thanks so much for the patience and the reply.  I am in awe of your hard work and dedication on this project, and aspire to be a programmer of the same persistence and knowledge someday!  Your situation of wanting to go deeper than learning some proprietary 3D engine that lasts for a few years is the same as mine, and I have found your project a great help already.  Thanks!
